build.gradle

```diff
@@ -361,8 +365,10 @@ subprojects {
         sign configurations.archives
     }
 
+    def publishGate = project.tasks.getByPath(':buildAndTestAll')
```

Comment:
Rather than getting the task by path, couldn't you just use the task directly? So drop this line, and on the following line do:
`buildAndTestAll.dependsOn build`